cmd/manager/main.go

```diff
@@ -35,18 +36,57 @@ import (
 )
 
 func main() {
-	watchNamespace := flag.String("namespace", "", "Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.")
+	var (
+		leaderElect                  bool
+		leaderElectLeaseDuration     int64
+		leaderElectResourceNamespace string
+		watchNamespace               string
+	)
+
+	flag.StringVar(
+		&watchNamespace,
+		"namespace",
+		"",
+		"Namespace that the controller watches to reconcile machine-api objects. If unspecified, the controller watches for machine-api objects across all namespaces.",
+	)
+
+	flag.StringVar(
+		&leaderElectResourceNamespace,
+		"leader-elect-resource-namespace",
+		"",
+		"The namespace of resource object that is used for locking during leader election. If unspecified, the controller watches for machine-api objects across all namespaces.",
+	)
+
+	flag.BoolVar(
+		&leaderElect,
+		"leader-elect",
+		true,
```

Comment:
Given there's no default namespace, and this would be a breaking change (technically), I think this should default to false

Reply:
Yes, I saw this happening locally. Will adjust the value.